internal/sirius/get_deputy_clients_test.go

```diff
@@ -595,3 +595,96 @@ func TestReformatCompletedDateReturnsNullIfNoDate(t *testing.T) {
 func TestReformatCompletedDateReturnsDateInCorrectFormat(t *testing.T) {
 	assert.Equal(t, "25/02/2020", reformatCompletedDate("2020-02-25T00:00:00+00:00"))
 }
+
+func TestGetOrderStatusReturnsOldestActiveOrder(t *testing.T) {
+	dateOne, _ := time.Parse("2006-02-01 00:00:00 +0000 UTC", "2014-12-01 00:00:00 +0000 UTC")
+	dateTwo, _ := time.Parse("2006-02-01 00:00:00 +0000 UTC", "2017-12-01 00:00:00 +0000 UTC")
+
+	orderData := Orders{
+		Order{OrderStatus: "Active", SupervisionLevel: "General", OrderDate: dateOne},
+		Order{OrderStatus: "Active", SupervisionLevel: "General", OrderDate: dateTwo},
+	}
+	expectedResponse := "Active"
```

Comment:
Would it be better add an assertion for the order date as well as the status the status is the same for both so it as it doesn't confirm with the oldest one was returned

Reply:
@hiboabd I think because getOrderStatus will only return an order status I don't think we can check the date (as it won't be returned). I will change the newer status to something else to prove it's returning active though.